src/common/components/data-display/LabeledRow.tsx

```diff
@@ -1,47 +1,91 @@
-import { Divider, Grid, makeStyles } from "@material-ui/core";
-import FileCopyOutlinedIcon from "@material-ui/icons/FileCopyOutlined";
+import { Grid, makeStyles } from "@material-ui/core";
 import React, { ReactElement } from "react";
 import { copyToClipboard } from "../../utils/appUtil";
-import IconButton from "../input/buttons/IconButton";
+import { copyIcon } from "../../../common/utils/svgIcons";
+import ButtonBase from "@material-ui/core/ButtonBase";
```

Comment:
There are button components in this project. Let's just use them in order to handle consistent style throughout the project.